src/EditorFeatures/Core/InlineRename/CommandHandlers/AbstractRenameCommandHandler_WordDeleteHandler.cs

```diff
@@ -34,6 +34,12 @@ private bool HandleWordDeleteCommand(ITextBuffer subjectBuffer, ITextView view,
             return false;
         }
 
+        if (renameService.ActiveSession.IsCommitInProgress)
+        {
+            // When rename commit is in progress, handle the command so it won't change the workspace
```

Comment:
i would change the terminology to 'swallow' to make it clear that we're both not processing it, nor letting anyone else proces sit.